EZSource/Classes/TableViewDataSource/SectionSource.swift

```diff
@@ -65,4 +65,20 @@ class SectionSource {
         self.sections = sections.replacingOccurrences(with: section, where: { $0.id == section.id })
     }
     
+    func deleteEmptySections() -> [DeleteSectionInfo] {
```

Comment:
Shouldn't this be public? I still wondered how it's accessible in Example

Reply:
I guess we resolved this during the discussion?